metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -1611,63 +1644,82 @@ def emit_published_datasources(self) -> Iterable[MetadataWorkUnit]:
-                    paths=[f"/{self.config.env.lower()}/{self.platform}/{browse_path}"]
-                )
-                dataset_snapshot.aspects.append(browse_paths)
-            else:
-                logger.debug(f"Browse path not set for table {table_urn}")
-            schema_metadata = None
-            if columns:
-                fields = []
-                for field in columns:
-                    if field.get(tableau_constant.NAME) is None:
-                        logger.warning(
-                            f"Skipping field {field[tableau_constant.ID]} from schema since its name is none"
-                        )
-                        continue
-                    nativeDataType = field.get(
-                        tableau_constant.REMOTE_TYPE, tableau_constant.UNKNOWN
-                    )
-                    TypeClass = FIELD_TYPE_MAPPING.get(nativeDataType, NullTypeClass)
+            return
 
-                    schema_field = SchemaField(
-                        fieldPath=field[tableau_constant.NAME],
-                        type=SchemaFieldDataType(type=TypeClass()),
-                        description="",
-                        nativeDataType=nativeDataType,
+        dataset_snapshot = DatasetSnapshot(
+            urn=database_table.urn,
+            aspects=[],
+        )
+        if database_table.paths:
+            # Browse path
+            browse_paths = BrowsePathsClass(
+                paths=[
+                    f"/{self.config.env.lower()}/{self.platform}/{path}"
+                    for path in database_table.paths
+                ]
+            )
+            dataset_snapshot.aspects.append(browse_paths)
+        else:
+            logger.debug(f"Browse path not set for table {database_table.urn}")
+        schema_metadata = None
+        if columns:
+            fields = []
+            for field in columns:
+                if field.get(tableau_constant.NAME) is None:
+                    logger.warning(
+                        f"Skipping field {field[tableau_constant.ID]} from schema since its name is none"
                     )
+                    continue
+                nativeDataType = field.get(
+                    tableau_constant.REMOTE_TYPE, tableau_constant.UNKNOWN
+                )
+                TypeClass = FIELD_TYPE_MAPPING.get(nativeDataType, NullTypeClass)
 
-                    fields.append(schema_field)
-
-                schema_metadata = SchemaMetadata(
-                    schemaName="test",
-                    platform=f"urn:li:dataPlatform:{self.platform}",
-                    version=0,
-                    fields=fields,
-                    hash="",
-                    platformSchema=OtherSchema(rawSchema=""),
+                schema_field = SchemaField(
+                    fieldPath=field[tableau_constant.NAME],
+                    type=SchemaFieldDataType(type=TypeClass()),
+                    description="",
```

Comment:
Is description a non-nullable field otherwise, I think omitting it would be better.